index.js

```diff
@@ -34,15 +34,45 @@
   ) {
     HTMLElement.prototype.nativeFocus = HTMLElement.prototype.focus;
 
+    var calcScrollableElements = function(element) {
+      var parent = element.parentNode;
+      var scrollableElements = [];
+
+      while (parent && parent !== document.scrollingElement) {
+        if (
+          parent.offsetHeight < parent.scrollHeight ||
+          parent.offsetWidth < parent.scrollWidth
+        ) {
+          scrollableElements.push([
+            parent,
+            parent.scrollTop,
+            parent.scrollLeft
+          ]);
+        }
+        parent = parent.parentNode;
+      }
+      parent = document.scrollingElement;
+      scrollableElements.push([parent, parent.scrollTop, parent.scrollLeft]);
```

Comment:
FYI to anyone in the future that if `document.scrollingElement` doesn't exist, this will throw an error. Example would be in Internet Explorer or when testing with `jest` or anything that uses `jsdom`.

@calvellido As `document.scrollingElement` is `document.documentElement` by default (https://developer.mozilla.org/en-US/docs/Web/API/document/scrollingElement), would it make sense to fall back to `documentElement` when `scrollingElement` is not found?
```js
parent = document.scrollingElement || document.documentElement
```

* * *

For `jest` users, place this in your test setup file:
```js
document.scrollingElement = document.documentElement
```